chalicelib/checks/wfr_encode_checks.py

```diff
@@ -0,0 +1,490 @@
+                    # xcor does not have qc, use ta indexes to find the correct files
+                    for ta_f in ta_2:
+                        taxcor_2.append(taxcor[ta.index(ta_f)])
+                    ta = ta_2
+                    taxcor = taxcor_2
+                    # for control files ,also select best2
+                    ta_cnt = wfr_utils.select_best_2(ta_cnt, all_files, all_qcs)
+
+                # collect step2 input files
+                s2_input_files = {}
+                if organism == 'human':
+                    org = 'hs'
+                    s2_input_files['chip.blacklist'] = '/files-reference/4DNFIZ1TGJZR/'
+                    s2_input_files['chip.chrsz'] = '/files-reference/4DNFIZJB62D1/'
+                if organism == 'mouse':
+                    org = 'mm'
+                    s2_input_files['chip.blacklist'] = '/files-reference/4DNFIZ3FBPK8/'
+                    s2_input_files['chip.chrsz'] = '/files-reference/4DNFIBP173GC/'
+
+                def rename_chip(input_at_id_list):
+                    # rename bed.gz to tagAlign.gz
+                    renamed = []
+                    for a_file in input_at_id_list:
+                        acc = a_file.split('/')[2]
+                        renamed.append(acc + '.tagAlign.gz')
+                    return renamed
+
+                s2_input_files['additional_file_parameters'] = {}
+                s2_input_files['chip.tas'] = ta
+                s2_input_files['additional_file_parameters']['chip.tas'] = {"rename": rename_chip(ta)}
+                s2_input_files['chip.bam2ta_no_filt_R1.ta'] = taxcor
+                s2_input_files['additional_file_parameters']['chip.bam2ta_no_filt_R1.ta'] = {"rename": rename_chip(taxcor)}
+                if ta_cnt:
+                    s2_input_files['chip.ctl_tas'] = ta_cnt
+                    s2_input_files['additional_file_parameters']['chip.ctl_tas'] = {"rename": rename_chip(ta_cnt)}
+
+                # collect parameters
+                parameters = {}
+                if paired == 'single':
+                    chip_p = False
+                elif paired == 'paired':
+                    chip_p = True
+                if not control_set:
+                    if target_type == 'histone':
+                        set_summary += "| skipped - histone without control needs attention, ie change to tf"
+                        check.brief_output.append(set_summary)
+                        check.full_output['skipped'].append({set_acc: set_summary})
+                        continue
+                run_ids = {'run_name': set_acc,
+                           'desc': a_set.get('description', '')}
+                parameters = {
+                    "chip.pipeline_type": target_type,
+                    "chip.paired_end": chip_p,
+                    "chip.choose_ctl.always_use_pooled_ctl": True,
+                    "chip.qc_report.name": run_ids['run_name'],
```

Comment:
There is an issue with the display_title of some of the ChIP-seq Quality Metric objects when using the "chip.qc_report.name” parameter. The calculated property takes the value from the 'name' field created by this parameter and displays it. We saw that some runs did not make use of this parameter and it worked fine. They also did not make use of the parameter "chip.qc_report.desc”. So, do not think we need to make use of these parameters.

Reply:
How did that get the name and description if not from the run parameters? Can you share an example for the ones that worked fine without the parameters?

If there are not used at all, feel free to take them out.